### PR TITLE
Rebase/element ios 1.9.14 #704

### PR DESCRIPTION
**Pull request by NicolasBuquet**
### Pull Request Checklist

- [x] I read the [contributing guide](https://github.com/tchapgouv/tchap-ios/blob/develop/CONTRIBUTING.md)
- [ ] UI change has been tested on both light and dark themes, in portrait and landscape orientations and on iPhone and iPad simulators
- [ ] Accessibility has been taken into account.
* [x] Pull request is based on the develop branch
- [ ] Pull request contains a [changelog file](https://github.com/matrix-org/matrix-ios-sdk/blob/develop/CONTRIBUTING.md#changelog) in ./changelog.d
- [x] You've made a self review of your PR
- [ ] Pull request includes screenshots or videos of UI changes
- [ ] Pull request includes a [sign off](https://github.com/matrix-org/matrix-ios-sdk/blob/develop/CONTRIBUTING.md#sign-off)

### CODE REVIEW

File: .github/workflows/triage-move-labelled.yml [hunk 1]
```diff
@@ -0,0 +1,313 @@
+name: Move labelled issues to correct boards and columns
```
Comment (phlniji) [inline .github/workflows/triage-move-labelled.yml:1]: All triage files are specific to Element.
So you should remove them from the .github/workflows folder.

Comment (NicolasBuquet) [reply .github/workflows/triage-move-labelled.yml]: Done.

File: Tchap/Config/BuildSettings.swift [hunk 2]
```diff
@@ -251,6 +251,8 @@ final class BuildSettings: NSObject {
     
     static let allowInviteExernalUsers: Bool = true
     
+    static let allowBackgroundAudioMessagePlayback: Bool = true
```
Comment (phlniji) [inline Tchap/Config/BuildSettings.swift:254]: Please add this change in Btchap and DevTchap `BuildSettings.swift` files too.

File: Tchap/Config/BuildSettings.swift [hunk 3]
```diff
@@ -425,6 +427,10 @@ final class BuildSettings: NSObject {
     
     static let locationSharingEnabled = false // Currently disabled in Tchap.
 
+    // MARK: - Voice Broadcast
+    static let voiceBroadcastChunkLength: Int = 120
+    static let voiceBroadcastMaxLength: UInt = 14400 // 240min.
+
```
Comment (phlniji) [inline Tchap/Config/BuildSettings.swift:433]: Please add this change in Btchap and DevTchap `BuildSettings.swift` files too.

File: Tchap/Config/BuildSettings.swift [hunk 4]
```diff
@@ -440,4 +446,18 @@ final class BuildSettings: NSObject {
...
+    static let qrLoginEnabledFromNotAuthenticated = true
+    /// Flag indicating whether the QR login enabled from Device Manager screen
+    static let qrLoginEnabledFromAuthenticated = false
+    /// Flag indicating whether displaying QRs enabled for the QR login screens
+    static let qrLoginEnableDisplayingQRs = false
+    
+    static let rendezvousServerBaseURL = URL(string: "https://rendezvous.lab.element.dev/")!
+
+    // MARK: - Alerts
+    static let showUnverifiedSessionsAlert = true
```
Comment (phlniji) [inline Tchap/Config/BuildSettings.swift:462]: Please add this change in Btchap and DevTchap `BuildSettings.swift` files too.

File: Riot/Modules/Room/RoomCoordinator.swift [hunk 5]
```diff
@@ -92,8 +92,9 @@ final class RoomCoordinator: NSObject, RoomCoordinatorProtocol {
         
         self.roomViewController.parentSpaceId = parameters.parentSpaceId
 
-        // Tchap: Disable Polls
-//        TimelinePollProvider.shared.session = parameters.session
+        TimelinePollProvider.shared.session = parameters.session
```
[the remark this answers is not in the document]
Comment (NicolasBuquet) [reply Riot/Modules/Room/RoomCoordinator.swift:95]: Done

File: Tchap/Config/BuildSettings.swift (hunk 2 above)
Comment (NicolasBuquet) [reply Tchap/Config/BuildSettings.swift:254]: Done

File: Tchap/Config/BuildSettings.swift (hunk 3 above)
Comment (NicolasBuquet) [reply Tchap/Config/BuildSettings.swift:433]: Done

File: Tchap/Config/BuildSettings.swift (hunk 4 above)
Comment (NicolasBuquet) [reply Tchap/Config/BuildSettings.swift:462]: Done

File: Riot/Modules/Settings/SignOut/SignOutFlowPresenter.swift [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+    
+    /// Starts the flow, presenting any popups on iPad from the specified view.
+    func start(sourceView: UIView?) {
+        guard let keyBackup = session.crypto?.backup else { return }
+        
+        signOutAlertPresenter.present(for: keyBackup.state,
+                                      areThereKeysToBackup: keyBackup.hasKeysToBackup,
+                                      from: presentingViewController,
+                                      sourceView: sourceView ?? presentingViewController.view,
+                                      animated: true)
```
Comment (phlniji) [inline Riot/Modules/Settings/SignOut/SignOutFlowPresenter.swift:62]: To fix the unable to SignOut issue, I suggest you this kind of change :
```suggestion
        // Tchap: Present the SignOut alert despite the lack of keyBackup (for now).
//        guard let keyBackup = session.crypto?.backup else { return }
        let keyBackup = session.crypto?.backup
        
        signOutAlertPresenter.present(for: keyBackup?.state ?? MXKeyBackupStateDisabled,
                                      areThereKeysToBackup: keyBackup?.hasKeysToBackup ?? false,
                                      from: presentingViewController,
                                      sourceView: sourceView ?? presentingViewController.view,
                                      animated: true)
```

Comment (NicolasBuquet) [reply Riot/Modules/Settings/SignOut/SignOutFlowPresenter.swift]: Done

**Review by phlniji — APPROVED** [review]
LGTM
Don't forget to squash the rebase resolution commits before merge (like explained in the rebase process document).

**Pull request merged**